Bearded.Utilities/Noise/NoiseMap.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace Bearded.Utilities.Noise
+{
+    public static class NoiseMap
+    {
+        public static INoiseMap FromArray(double[,] array, IInterpolationMethod2 interpolation) =>
+            new ArrayNoiseMap(array, interpolation);
+
+        private sealed class ArrayNoiseMap : INoiseMap
+        {
+            private readonly double[,] array;
+            private readonly IInterpolationMethod2 interpolation;
+            private readonly int width;
+            private readonly int height;
+
+            public ArrayNoiseMap(double[,] array, IInterpolationMethod2 interpolation)
+            {
+                this.array = array;
```

Comment:
Is it intended that we don't make a defensive copy? I.e. this class is supposed to be a wrapper around the mutable array?